addons/trenches/functions/fnc_initTrench.sqf

```diff
@@ -0,0 +1,35 @@
+#include "script_component.hpp"
+/*
+ * Author: chris579, Salbei
+ * Inits a trench
+ *
+ * Arguments:
+ * 0: trench <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [TrenchObj] call ace_trenches_fnc_initTrench
+ *
+ * Public: No
+ */
+
+params [
+    ["_object", objNull, [objNull]]
+];
+if (isServer) then {
+   _object setVariable [QGVAR(progress), 1, true];
```

Comment:
Just as a side note, can you remove variables from JIP queue by setting them to nil with PVAR flag set? 
For variables that don't need to be in JIP. Would be neat to remove some things when trench is done placing, ACE is already shitting over JIP queue, no need to make it even worse :D

Reply:
Well it is needed for JIPs, what if you wnat to remoe the trench mid mission?